### PR TITLE
Fix unpaid items are counted

### PR DESCRIPTION
**Pull request by ekzyis**
Fix this:

https://github.com/user-attachments/assets/2c6ecc00-5b8e-4401-841c-fab7b57be939

Tested with anon and logged in. Also tested with items that were paid with fee credits (`invoiceActionState IS NULL`).

### CODE REVIEW

**Comment by ekzyis** [comment]
I just searched for `models.item.count` and also fixed it for `ncomments`. I also noticed that `auctionPosition` also calls `models.item.count` but it already uses `item.status` which is enough afaict.

File: api/resolvers/user.js [hunk 1]
```diff
@@ -922,11 +922,12 @@ export default {
           createdAt: {
             gte,
             lte
-          }
+          },
+          ...(me?.id === user.id ? {} : { OR: [{ invoiceActionState: 'PAID' }, { invoiceActionState: null }] })
```
Comment (huumn) [inline api/resolvers/user.js:926]: This can all be an `OR` I think:

e.g.

```
OR: [{userId: me.id}, { invoiceActionState: 'PAID' }, { invoiceActionState: null }]
```

Alternatively, we don't need to count unpaid items at all and it can just be:

```
OR: [{ invoiceActionState: 'PAID' }, { invoiceActionState: null }]
```

Comment (ekzyis) [reply api/resolvers/user.js]: Decided to never count unpaid items
